hq/app/logic/ReportDisplay.scala

```diff
@@ -91,6 +92,17 @@ object ReportDisplay {
     ReportSummary(warnings, errors, other)
   }
 
+  def exposedKeysSummary(allReports: Map[AwsAccount, Either[FailedAttempt, List[ExposedIAMKeyDetail]]]): Map[AwsAccount, Boolean] = {
+    def checkForExposedKeys(temp: Either[FailedAttempt, List[ExposedIAMKeyDetail]]): Boolean = temp match {
```

Comment:
I don't think you meant to leave this named as temp?

Also, does this need to be a def? It's only called once so probably better inline.